playbooks/vars/maas-tigkstack.yml

```diff
@@ -31,6 +31,11 @@ maas_grafana_key:
 maas_grafana_repo:
   url: "deb https://packagecloud.io/grafana/stable/debian/ jessie main"
 
+#
+# Set the desired package state (present/latest)
+#
+maas_tigkstack_apt_package_state: present
```

Comment:
The current state is `latest`, and everything else in this role seems to use `latest`.

Reply:
Also, you may want to just re-use `maas_package_state` here, rather than introduce a new var?  Looks like all other apt package sets are installed with `maas_package_state`.

Reply:
I used present, because the lint rule doesn't allow latest. However I didn't notice the package state var, I'll flip over to that.

Reply:
@hughsaunders I fixed these yesterday, if you rebase you'll get present over latest

Reply:
Will do.